linera-chain/src/chain.rs

```diff
@@ -32,9 +32,21 @@ use linera_views::{
     set_view::SetView,
     views::{CryptoHashView, GraphQLView, RootView, View, ViewError},
 };
+use once_cell::sync::Lazy;
+use prometheus::{register_int_counter_vec, IntCounterVec};
 use serde::{Deserialize, Serialize};
 use std::collections::{BTreeMap, HashSet};
 
+pub static NUM_BLOCKS_EXECUTED: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
+        "num_blocks_executed",
+        "Number of blocks executed",
+        // Can add labels here
+        &[]
```

Comment:
This is where I mentioned we can add labels like `chain_id` or whatever else we want

Reply:
This probably needs to be a reasonably-small static set of labels for easy aggregation. We can probably come up with special categories of chains later on.